### PR TITLE
Clean the Java native implementation 

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Review by niveathika — COMMENTED** [review]
Some of these are used in SQL libraries. Will this break those APIs?

**Comment by BuddhiWathsala** [comment]
> Some of these are used in SQL libraries. Will this break those APIs?

The time APIs that uses in other stdlibs are as follows:

- file - `TimeValueHandler.createUtcFromMilliSeconds()`
- postgressql - `TimeValueHandler.createCivilFromZoneDateTimeString()`
- postgressql - `TimeValueHandler.createZonedDateTimeFromUtc()`
- crypto - `TimeValueHandler.createUtcFromMilliSeconds()`
- sql - `TimeValueHandler.createUtcFromMilliSeconds()`
- sql - `TimeValueHandler.createZonedDateTimeFromUtc()`

The behaviour of the abovementioned APIs didn’t change in this PR.

File: native/src/main/java/io/ballerina/stdlib/time/nativeimpl/Civil.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    private BMap<BString, Object> civilMap = ValueCreator.createRecordValue(ModuleUtils.getModule(),
+            Constants.CIVIL_RECORD);
+
+    public BMap<BString, Object> buildFromZonedDateTime(ZonedDateTime zonedDateTime) {
+
+        this.zonedDateTime = zonedDateTime;
+        setCommonCivilFields();
+        BigDecimal second = new BigDecimal(zonedDateTime.getSecond());
+        second = second.add(new BigDecimal(zonedDateTime.getNano()).divide(ANALOG_GIGA, MathContext.DECIMAL128));
+        civilMap.put(StringUtils.fromString(Constants.TIME_OF_DAY_RECORD_SECOND),
```
Comment (daneshk) [inline native/src/main/java/io/ballerina/stdlib/time/nativeimpl/Civil.java:54]: Shall we move `StringUtils.fromString(Constants.TIME_OF_DAY_RECORD_SECOND)` as constant and refer it here. So we don't need to create BString everytime

Comment (BuddhiWathsala) [reply native/src/main/java/io/ballerina/stdlib/time/nativeimpl/Civil.java]: Fixed in https://github.com/ballerina-platform/module-ballerina-time/pull/326/commits/95fe21f93ca0926a0586758dc59abcf18df4f4ab

**Review by daneshk — APPROVED** [review]

**Pull request merged**